test/fixtures/relationships.yml

```diff
@@ -1,16 +1,16 @@
 # Relationships
```

Comment:
once again, may as well just remove this file if they are all commented out and no longer needed